### PR TITLE
BUGFIX: Creating thumbnails on asset creation

### PR DESCRIPTION
**Pull request by aertmann**
Merge regression of d03079cb90fbade6fae5748c0dd9acb94543c761

### CODE REVIEW

File: TYPO3.Media/Classes/TYPO3/Media/Package.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 use TYPO3\Flow\Core\Bootstrap;
 use TYPO3\Flow\Package\Package as BasePackage;
-use TYPO3\Media\Domain\Model\Asset;
+use TYPO3\Media\Domain\Service\AssetService;
```
Comment (aertmann) [inline TYPO3.Media/Classes/TYPO3/Media/Package.php]: don't get why it's not throwing a fatal error 🤔

Comment (kdambekalns) [inline TYPO3.Media/Classes/TYPO3/Media/Package.php]: Well, is the import used? Otherwise PHP just doesn't care AFAIK.

Comment (kdambekalns) [inline TYPO3.Media/Classes/TYPO3/Media/Package.php]: Ok, it is used, but in the `connect()` call will resolve to a string just fine. Only the connectd signal will never fire… :)

Comment (aertmann) [inline TYPO3.Media/Classes/TYPO3/Media/Package.php]: oh didn't expect it not to fail from a missing class name, good to konw

**Comment by aertmann** [comment]
Thanks for reviewing 👍

File: TYPO3.Media/Classes/TYPO3/Media/Package.php (hunk 1 above)
Comment (kdambekalns) [inline TYPO3.Media/Classes/TYPO3/Media/Package.php]: Actually a good thing, think `class_exists(Some\Class::class)`…

**Comment by kdambekalns** [comment]
🎶  Don't worry about a test…

**Pull request merged**